phoenix-core/src/main/java/org/apache/phoenix/mapreduce/util/PhoenixConfigurationUtil.java

```diff
@@ -866,4 +872,18 @@ public static void setTenantId(Configuration configuration, String tenantId){
         configuration.set(MAPREDUCE_TENANT_ID, tenantId);
     }
 
+    public static void setMRSnapshotManagedInternally(Configuration configuration, Boolean isSnapshotRestoreManagedInternally) {
+        Preconditions.checkNotNull(configuration);
+        Preconditions.checkNotNull(isSnapshotRestoreManagedInternally);
+        configuration.set(MANAGE_MR_SNAPSHOT_RESTORE_INTERNALLY,
+                String.valueOf(isSnapshotRestoreManagedInternally));
+    }
+
+    public static boolean getMRSnapshotManagedInternally(final Configuration configuration) {
```

Comment:
In addition to switching the setting to Externally, this should probably have an "is" prefix, not a get, because it's a boolean. (As you do for the local variable)